web/server/sitemap/sitemap.ts

```diff
@@ -5,19 +5,13 @@ import { template } from './sitemap.template';
 import { Manifest, Url } from './types';
 
 export const generateSiteMapXML = (options: { manifest: Manifest; staticDir: string; host: string }) => {
-  if (isDevelopment()) return;
```

Comment:
@vdfdev I am adding back this restriction because on `development` there is no `app.pagesManifest`. I don't know why it doesn't exist. But without that guard, the app breaks when running on dev.

Reply:
Sure! Sorry about that, I should have tested...